src/components/organisms/sidebar/index.tsx

```diff
@@ -1,16 +1,16 @@
 import { useAdminStore } from "medusa-react"
```

Comment:
todo(this-file): Sorry, this one is on me, I should have spotted this in the design; "Orders" should be kept at the top, and "Promotions" should remain as "Discounts". The latter must be a relic from older times when we contemplated making the switch.

Reply:
Reverted.